### DISCUSSION TITLE
Adding members to group by showing list

### DISCUSSION THREAD

**Discussion by Xitija** [Sunbird-Ed/Community#561] [Sunbird ED configuration discussion] [answered]
Hello All, 

I am trying to implement a dropdown to select members and add them to group.
I am planning to bring a dropdown on the following page
![addnewdropdown](https://github.com/Sunbird-Ed/Community/assets/65657373/08e9fb6b-a9a5-49ea-8826-dbe55facb18e)

I want to populate a list of users in the dropdown whom I would select and add them to the group.

Can anyone guide me on how this can be done?

cc: @coolbung @Renuka-S @komalm

**Comment by coolbung** [comment]
A few things to consider

1. UX - Will a dropdown work if there are hundreds or thousands of users ? Maybe a type-ahead search might be better
2. Privacy - Which users can come in the dropdown ? From the same org / from all orgs ? Ideally this needs to be a org level configuration

For the implementation itself, you can integrate the user search API to get a list of users to show

**Reply by Krishnaj20** [reply]
@Xitija - hope your query was responded by Ashwin. 
@amiableanil - can you confirm the max number of users who can be added to a group? Is this a config?

**Reply by AmiableAnil** [reply, marked as answer]
@Xitija This info is available in microsite. Please find it here. https://lern.sunbird.org/learn/product-and-developer-guide/groups/functional-configurations
